Screen.cpp

```diff
@@ -0,0 +1,97 @@
+#include "Screen.h"
+
+LiquidCrystal_I2C lcd(0x27,20,4);
+
+Screen::Screen(){}
+
+void Screen::calibrationAll() {
+  bno.begin();
+  bno.setExtCrystalUse(true);
+  lcd.init();
+  lcd.backlight();
+  writeLCDdown("LCD ready");
+  lcd.clear();
+  lcd.setCursor(0,0);
+  
+  while(bno_.orientationStatus() != 3) {
+    lcd.setCursor(6,0);
+    lcd.print("I´m not ready");
+  }
+  lcd.setCursor(6,1);
+  lcd.print("I´m ready");
+
+  delay(kDelayAfterBNO);
+
+  pinMode(LED, INPUT); 
+  digitalWrite(LED, LOW);
+  
+  pinMode(motors.kMotorLeftForward1, OUTPUT);
+  pinMode(motors.kMotorLeftForward2, OUTPUT);
+  pinMode(motors.kMotorLeftBack1, OUTPUT);
+  pinMode(motors.kMotorLeftBack2, OUTPUT);
+  pinMode(motors.kMotorRightForward1, OUTPUT);
+  pinMode(motors.kMotorRightForward2, OUTPUT);
+  pinMode(motors.kMotorRightBack1, OUTPUT);
+  pinMode(motors.kMotorRightBack2, OUTPUT);
+
+  sensors_event_t event;
+  bno.getEvent(&event);
+}
```

Comment:
I'd separate this into several functions.

Try and move everything bno related to only the BNO file. The goal is that you can import the BNO file to any arduino code and use the bno properly. Same goal with the Screen file, use the lcd properly. Leave them only to the components they should be.